pkgs/applications/networking/instant-messengers/pidgin-plugins/purple-facebook/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, pidgin, glib, json_glib } :
+
+stdenv.mkDerivation rec {
+  name = "purple-facebook-${version}";
+  version = "66ee77378d82";
```

Comment:
But do note that this *is* a release version, so the clause "if the package is not a release but a commit" doesn't apply. The version parsing issue remains, of course.